index.html

```diff
@@ -51,13 +51,13 @@ <h1 class="welcome-title">Welcome to Real Dev Squad</h1>
           <h3>Check out our Docs</h3>
         </div>
         <div class="cards">
-          <div class="card">
+          <div id="card-1" class="card">
             <a href="./faq.html">FAQ</a>
           </div>
-          <div class="card">
+          <div id="card-2" class="card">
             <a href="./discord.html">Discord</a>
           </div>
-          <div class="card">
+          <div id="card-3" class="card">
             <a href="./code-of-conduct.html">Code of Conduct</a>
           </div>
         </div>
```

Comment:
it can be rewritten something like this

```
<div class="cards" id="cards">
    <div class="card" data-url="https://welcome.realdevsquad.com/faq.html">
      <a>FAQ</a>
    </div>
    <div class="card" data-url="https://welcome.realdevsquad.com/discord.html">
      <a>Discord</a>
    </div>
    <div class="card" data-url="https://welcome.realdevsquad.com/code-of-conduct.html">
      <a>Code of Conduct</a>
    </div>
</div>
```

Here, we are adding event listeners only to the parent node.

Also, you can extend it whenever required by just adding a new card will work fine.

Reply:
Even all the URLs should be written in a separate `constant.js` file and must be imported wherever required.